src/go/rpk/pkg/cli/cmd/redpanda/admin/admin.go

```diff
@@ -0,0 +1,71 @@
+// Package admin provides a cobra command for the redpanda admin listener.
```

Comment:
Please add the license header in this file and other new files:

```
// Copyright 2021 Vectorized, Inc.
//
// Use of this software is governed by the Business Source License
// included in the file licenses/BSL.md
//
// As of the Change Date specified in that file, in accordance with
// the Business Source License, use of this software will be governed
// by the Apache License, Version 2.0
```

We need to automate this check...

Reply:
I was hoping this actually was not needed, but [this SO comment](https://stackoverflow.com/questions/845895/putting-license-in-each-code-file#comment654481_845904) is compelling enough.

There exists a lint for this header check! I think we can port the linters from the linter proposal/other repo over to rpk eventually (soon), and add the header linter for specifically this.

Reply:
Done.